README.md

```diff
@@ -7,7 +7,7 @@ Basic express server using modularization, middleware and tests and deploying to
 ## heroku
 
 - [heroku-deployment](https://tahmina-basic-server.herokuapp.com/)
-[heroku-activity-deployment](https://dashboard.heroku.com/apps/tahmina-basic-server/activity)
+- [heroku-activity-deployment](https://dashboard.heroku.com/apps/tahmina-basic-server/activity)
 
 ## Github
```

Comment:
With future labs I would love to see more added to your READMEs such as:
- UML image
- .env Requirement section
- How to run the project
- How to run tests that you have written